recipe/conda_build_config.yaml

```diff
@@ -1,30 +1,35 @@
-c_compiler:
-  - clang_bootstrap  # [osx]
-  - gcc              # [linux]
-  - vs2019           # [win]
-cxx_compiler:
-  - clang_bootstrap  # [osx]
-  - gxx              # [linux]
-  - vs2019           # [win]
-vc:
-  - 14
+c_compiler:           # [osx]
+  - clang_bootstrap   # [osx]
+cxx_compiler:         # [osx]
+  - clang_bootstrap   # [osx]
 
-c_compiler_version:     # [osx and x86_64]
-  - "12.0.0"            # [osx and x86_64]
-cxx_compiler_version:   # [osx and x86_64]
-  - "12.0.0"            # [osx and x86_64]
 # libcxx only supports the last released GCC version, see
-# https://releases.llvm.org/13.0.0/projects/libcxx/docs/index.html#platform-and-compiler-support
-c_compiler_version:     # [linux]
-  - 11                  # [linux]
-cxx_compiler_version:   # [linux]
-  - 11                  # [linux]
-
-MACOSX_SDK_VERSION:  # [osx and x86_64]
-  - 10.14            # [osx and x86_64]
-CONDA_BUILD_SYSROOT:  # [osx and x86_64]
-  - /opt/MacOSX10.14.sdk        # [osx and x86_64]
+# https://libcxx.llvm.org/index.html#platform-and-compiler-support
+c_compiler_version:
+  - 11.2                # [linux]
+  - 17                  # [osx]
+cxx_compiler_version:
+  - 11.2                # [linux]
+  - 17                  # [osx]
 
 dummy_var:              # [osx]
   - 0                   # [osx and x86_64]
   - 1                   # [osx and arm64]
+
+# We don't have this in defaults yet either for linux-64:
+# sysroot_linux-64                2.17     h57e8cba_10  pkgs/main
+# this should be updated to the latests 2.x when that work is done.
+c_stdlib_version:           # [linux]
+  # actually would need >=2.24, but we don't have that in conda-forge yet
+  - "2.17"                  # [linux]
+  # w.r.t. to the ABI of the system libcxx, this would need to be 12.0 on macos
+  # but as we only publish one build to main, maintain global lower bounds
+
+# conda-forge built a version both for >12 and <12, we're going to constrain this as >=12
+# so we can avoid building 2 versions and patching everything.
+# References:
+# https://github.com/conda-forge/libcxx-feedstock/blob/17.x/recipe/conda_build_config.yaml#L28-L33
+# https://github.com/conda-forge/libcxx-feedstock/blob/17.x/recipe/meta.yaml#L17-L18
+macos_min_version:
+  - 10.15 # [osx and x86_64]
+  - 12    # [osx and arm64]
```

Comment:
I don't understand this part. We leave 10.15 for intel and set 12 for arm64. Why can't we leave 11.3?

On the conda-forge recipe, I don't see what makes the build different between the two variants, except the pre-12 variant having __osx <12 and being published to a special channel. The post-12 variant does not show an os restriction.